src/graph.rs

```diff
@@ -1062,8 +1019,8 @@ impl PyGraph {
     ///     the graph
     #[pyo3(text_signature = "(self, index_list, /)")]
     pub fn remove_nodes_from(&mut self, index_list: Vec<usize>) -> PyResult<()> {
-        for node in index_list.iter().map(|x| NodeIndex::new(*x)) {
-            self.graph.remove_node(node);
+        for node in index_list {
+            self.remove_node(node)?;
```

Comment:
Why all the changes from `self.graph.remove_node` to `self.remove_node`?

Reply:
This is the second order issue (which the release note is about) where we weren't tracking removals correctly in several methods. Using `self.remove_node` does the tracking for us.